schema/me/conversation/message.js

```diff
@@ -44,7 +44,10 @@ export const MessageType = new GraphQLObjectType({
     is_invoice: {
       description: "True if message is an invoice message",
       type: GraphQLBoolean,
-      resolve: ({ metadata: { lewitt_invoice_id } }) => isExisty(lewitt_invoice_id),
+      resolve: ({ metadata }) => {
+        if (!metadata) return false
+        return isExisty(metadata.lewitt_invoice_id)
+      },
```

Comment:
Could make this a bit more idiomatic with `!!metadata && isExisty(metadata.lewitt_invoice_id)`.